.github/workflows/main.yml

```diff
@@ -2,11 +2,9 @@ name: CI
 
 on:
   push:
-    branches:
-      - 13_4
   pull_request:
     branches:
-      - 13_4
+      - BABEL_2_X_DEV__PG_14_2
```

Comment:
Please remove this line; the point of the commit was to stop all filtering of the workflow.

Reply:
okay, will update it.

Reply:
This line is still showing up. I think the correct thing to do is simply remove this commit entirely.

Reply:
Wait, I take that back. We still need the commit (to remove the 13_4 bit). But the commit should not add `BABEL_2_X_DEV__PG_14_2`, unless I'm missing something?

Reply:
I've verified this ultimately gets resolved by another commit at the end of this PR.